modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3212,6 +3212,21 @@
 		"issueKey": "LPD-35897",
 		"to": "getPreferences(param#0#)"
 	},
+	{
+		"classNames": [
+			"DLUtil"
+		],
+		"from": "regex:(DLUtil)\\.(?<methodName>getDownloadURL|getImagePreviewURL)",
```

Comment:
Hi @rubenssoares-br, since it's only a utility class the `classNames` aren't necessary. 
And the () isn't required.

		"from": "regex:DLUtil\\.(?<methodName>getDownloadURL|getImagePreviewURL)",

Reply:
Have you been able to check if both methods remain with the same parameter list?

Reply:
> Have you been able to check if both methods remain with the same parameter list?

Yes, they remain with the same list of parameters.